HW_3/task_1.py

```diff
@@ -0,0 +1,12 @@
+# Дан список повторяющихся элементов. Вернуть список с дублирующимися элементами.
+# В результирующем списке не должно быть дубликатов.
+
+
+my_list = [2, 4, 8, True, "qwe", 2, 9, "qwe", False, 1, "rty", 7, 2, True]
+temp = []
+for item in my_list:
```

Comment:
Чтобы немного  уменьшить сложность, используйте здесь множество.
```suggestion
for item in set(my_list):
```